src/AvoStorage.ts

```diff
@@ -5,53 +5,129 @@ export class AvoStorage {
   AsyncStorage: any | null = null;
 
   shouldLog: boolean;
-  initialized = false;
-  onInitFuncs: Array<() => void> = [];
+  storageInitialized = false;
+  itemsFromLastSessionLoaded = false;
+  useFallback = false;
+  fallbackStorage: any = {};
```

Comment:
Should we type this as an object with keys and values as strings?
```suggestion
  fallbackStorage: {[key: string]: string} = {};
```

Reply:
I think the value should be `string | null`